benchmarks/benchmark/tools/locust-load-inference/manifest-templates/locust-worker-controller.yaml.tpl

```diff
@@ -44,8 +44,10 @@ spec:
               value: ${tokenizer}
             - name: USE_BEAM_SEARCH
```

Comment:
Oh interesting, I'm surprised we don't need the secret volume mount like we do in TGI: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/7c5ca2808c502c7d63cc0d7ebe05957f95e967c2/benchmarks/inference-server/text-generation-inference/manifest-templates/text-generation-inference.tftpl#L54

do you know why that is? Maybe in a separate PR we can clean up the TGI implementation if it's not required

Reply:
AFAICT, tgi supplies the secret twice (once in an env var, again in /var/secrets) though not being familiar enough with tgi, I don't know why. My guess is that the volume mount is not used, but we'd need to try it out.